processor/signoztailsampler/internal/sampling/always_unsample.go

```diff
@@ -0,0 +1,25 @@
+package sampling
+
+import (
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.uber.org/zap"
+)
+
+type alwaysUnSample struct {
```

Comment:
nit:
```suggestion
type neverSample struct {
```

Reply:
Not sure I want to change the name and spend time on testing this. I can schedule this as time permits. Not committing the change for now.

Reply:
this is done.